app/services/host_inventory_api.rb

```diff
@@ -5,23 +5,22 @@
 
 # Interact with the Insights Host Inventory. Usually HTTP calls
 # are all that's needed.
-class HostInventoryAPI
+class HostInventoryApi
   # Error to raise if the inventory host could not be found
   class InventoryHostNotFound < StandardError; end
 
   ERRORS = [InventoryHostNotFound].freeze
 
-  def initialize(account, url, b64_identity)
+  def initialize(account: nil, url: Settings.host_inventory_url,
+                 b64_identity: account&.b64_identity)
     @url = "#{URI.parse(url)}#{Settings.path_prefix}/inventory/v1/hosts"
     @account = account
-    @b64_identity = b64_identity || @account.b64_identity
+    @b64_identity = b64_identity
```

Comment:
This introduced a regression. See https://github.com/RedHatInsights/compliance-backend/pull/727